Code/Engine/Foundation/Containers/Bitfield.h

```diff
@@ -109,6 +112,9 @@ class ezStaticBitfield
   /// \brief Clears the given bit to 0.
   void ClearBit(ezUInt32 uiBit); // [tested]
 
+  /// \brief Sets the given bit to 1 or 0 depending on the given value.
+  void SetBitValue(ezUInt32 uiBit, bool bValue); // [tested]
+
```

Comment:
duplicate code ??

Reply:
As discussed offline there are two versions of bitfield, ezBitfield and ezStaticBitfield both in the same file. So yes the code is duplicated but in different classes.